posts/2022-05-28-liberty-deep-dive.adoc

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Introducing a hands-on technical deep dive on Liberty!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Introducing a hands-on technical deep dive on Liberty! - OpenLiberty.io
+seo-description: The Liberty Deep Dive will teach you how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, document your REST API, persist data to a database, secure your applications using JSON Web Tokens (JWT), test your applications and more.
+blog_description: "TThe Liberty Deep Dive will teach you how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, document your REST API, persist data to a database, secure your applications using JSON Web Tokens (JWT), test your applications and more."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Introducing a hands-on technical deep dive on Liberty!
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+We are excited to announce the release of our newest tutorial, A Technical Deep Dive on Liberty! Our team have been hard at work to produce a single tutorial which covers many cloud-native concepts and bundles them into one application. This tutorial is aimed to help developers, from Liberty novices to those more experienced, develop, build, test, and deploy cloud-native Java applications and microservices on Open Liberty. You will learn how to build a RESTful microservice on Liberty with Jakarta EE and MicroProfile, documenting your REST API, persisting data to a database, securing your applications using JSON Web Tokens (JWT), testing your applications and more. You will use Maven throughout this exercise to build the microservice and to interact with the running Liberty instance. Lastly, you’ll build a container image for the microservice and deploy it to Kubernetes using the Open Liberty Operator.
+
+You can access the tutorial directly link:{url-prefix}/guides/liberty-deep-dive.html[through this link].
+
+Excitingly, this lab is also available for you to try in our cloud-hosted environment, without having to worry about local setup and prerequisites. Simply head over to the link:{url-prefix}/guides/liberty-deep-dive.html[tutorial page], and click the green *'Run in cloud'* button. Alternatively, you can head over to the workshop via link:https://ibm.biz/LibertyDeepDive[https://ibm.biz/LibertyDeepDive].
+
+image::/img/blog/deepdive.png[screen capture of the liberty deep dive UI,width=70%,align="center"]
+
+== Your feedback is welcomed
+
+Tell us about your experience with our guides by responding to the questions in the *Where to next?* section at the end of the tutorial. We’d love to hear from you!
```

Comment:
```suggestion
Tell us about your experience with this Technical Deep Dive on Liberty by responding to the questions in the *Where to next?* section at the end of the tutorial. We’d love to hear from you!
```

Reply:
Can we include a screen capture of the feedback section if not provide a direct link to the corresponding GH issue for users to file issues?

Reply:
I think we need to be consistent on the title of the tutorial. So:

"Tell us about your experience with A Technical Deep Dive on Liberty by responding to the questions in the *Where to next?* section at the end of the tutorial. We’d love to hear from you!"